src/org/daisy/braille/utils/pef/TextHandler.java

```diff
@@ -158,7 +158,9 @@ public Builder options(Map<String, String> settings) {
                     duplex("true".equals(value.toLowerCase()));
                 } else if (KEY_DATE.equals(key)) {
                     try {
-                        date(DATE_FORMAT.parse(value));
+                        synchronized (DATE_FORMAT) {
```

Comment:
Blocks should be synchronized on `private final` fields. See https://rules.sonarsource.com/java/RSPEC-2445
`DATE_FORMAT` is `public`.
Please introduce a `private final static Object SYNCHRONIZER; ` or so

Reply:
Changed DATE_FORMAT to a private final variable.